src/main/resources/fabric.mod.json

```diff
@@ -28,9 +28,9 @@
   ],
 
   "depends": {
-    "fabricloader": ">=0.12.0",
-    "minecraft": ">=1.17",
-    "java": ">=16"
+    "fabricloader": ">=0.13.3",
+    "minecraft": "1.18.2",
```

Comment:
It would be nice if this was `>=1.18.2`, as the mod is very likely to keep working in future versions.